templates/nav.hbs

```diff
@@ -20,10 +20,8 @@
     <div class="nav__spacer" aria-hidden="true" data-order="{{_navOrder}}"></div>
   {{/each}}
 
-  <button name="drawer" class="btn-icon nav__btn nav__drawer-btn js-nav-drawer-btn is-position-{{_config._drawer._position}}" data-event="toggleDrawer" aria-expanded="false" aria-label="{{_globals._accessibility._ariaLabels.navigationDrawer}}" data-order="{{_globals._extensions._drawer._navOrder}}" data-tooltip-id="drawer">
-    {{#if _config._drawer._iconClass}}
-    <span class="icon {{_config._drawer._iconClass}}" aria-hidden="true"></span>
-    {{/if}}
+  <button name="drawer" class="btn-icon nav__btn nav__drawer-btn js-nav-drawer-btn is-position-{{#if _config._drawer._position}}{{_config._drawer._position}}{{/if}}{{#unless _config._drawer._position}}auto{{/unless}}" data-event="toggleDrawer" aria-expanded="false" aria-label="{{_globals._accessibility._ariaLabels.navigationDrawer}}" data-order="{{_globals._extensions._drawer._navOrder}}" data-tooltip-id="drawer">
+    <span class="icon {{#if _config._drawer._iconClass}}{{_config._drawer._iconClass}}{{/if}}{{#unless _config._drawer._iconClass}}icon-list{{/unless}}" aria-hidden="true"></span>
```

Comment:
```suggestion
    <span class="icon {{#if _config._drawer._iconClass}}{{_config._drawer._iconClass}}{{else}}icon-list{{/if}}" aria-hidden="true"></span>
```

https://handlebarsjs.com/guide/block-helpers.html#conditionals

You happy this this in the handlebars rather than the javascript?
https://github.com/adaptlearning/adapt-contrib-core/blob/7b126ae7bcce7ea22d2ecdf57069b4b0b2826feb/js/views/navigationView.js#L80

Reply:
> You happy this this in the handlebars rather than the javascript?

@oliverfoster I think so, yeah. It's just a stopgap solution, and it will be easy to remove once we use Navigation Button API.